qnexus/client/projects.py

```diff
@@ -154,7 +161,9 @@ def get_or_create(
 
 def _fetch_by_id(project_id: UUID | str) -> ProjectRef:
     """Utility method for fetching directly by a unique identifier."""
-    res = get_nexus_client().get(f"/api/projects/v1beta/{project_id}")
+    res = get_nexus_client().get(
+        f"/api/projects/v1beta/{project_id}?scope=global_admin"
+    )
```

Comment:
Same comment as before re `?scope=global_admin`

Reply:
Ah I'm sorry, indeed thats left over. Will fix the sloppy PR!